homeassistant/components/sensibo/select.py

```diff
@@ -90,15 +107,21 @@ def __init__(
     @property
     def current_option(self) -> str | None:
         """Return the current selected option."""
-        return self.entity_description.value_fn(self.device_data)
+        state = self.entity_description.value_fn(self.device_data)
+        if TYPE_CHECKING:
+            assert isinstance(state, str)
+        return state.lower()
 
     @property
     def options(self) -> list[str]:
         """Return possible options."""
         options = self.entity_description.options_fn(self.device_data)
         if TYPE_CHECKING:
             assert options is not None
-        return options
+        lower_case_options = []
+        for option in options:
+            lower_case_options.append(option.lower())
+        return lower_case_options
```

Comment:
Can we calculate this when we get an updated state, to avoid the transformation each time the property is accessed?